src/software/ai/passing/cost_function.cpp

```diff
@@ -258,8 +258,8 @@ double ratePassFriendlyCapability(Team friendly_team, const Pass& pass,
 
     // Figure out what time the robot would have to receive the ball at
     Duration ball_travel_time = Duration::fromSeconds(
-        (pass.receiverPoint() - pass.passerPoint()).length() / pass.speed());
-    Timestamp receive_time = pass.startTime() + ball_travel_time;
+        (pass.receiverPoint() - ball.position()).length() / pass.speed());
```

Comment:
I'd suggest putting this back to the passer point. Consider the case in the future where we dribble the ball to the passer point (possible breaking some ankles in the process) before kicking. Ideally we split the evaluation of the pass from the actual execution.

Reply:
yeah that makes a lot of sense, added back!